### PR TITLE
Week3 aditya

### PR DESCRIPTION
**Pull request by sirElvin**
here i've created the flashcard.tsx using useDictionary hook

### CODE REVIEW

**Comment by binamkayastha** [comment]
Made the following changes:

- Merged the main branch into this branch
- Fixed all the lint and build issues
- Updated the Header component with the new NepaLingo logo
- Added a FlashcardPage component which includes the new header

File: nepalingo-web/src/components/Card.tsx [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
+  Word,
+  TranslatedWord,
+  Pronunciation,
+  DevenagiriSpelling,
+  ImageUrl,
+  PronounciationUrl,
+  isFlipped,
+}) => {
+  const [audio, setAudio] = useState<HTMLAudioElement | null>(null);
+  console.log(ImageUrl)
```
Comment (christikaes) [inline nepalingo-web/src/components/Card.tsx:25]: can we please remove the log statements?

File: nepalingo-web/src/components/Header.tsx [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
-      <header className={styles.header}>
-        <h1 className="text-3xl font-bold underline">Hello, World!</h1>
-      </header>
-    </div>
+    <nav className="bg-white border-gray-200 px-4 lg:px-6 py-2.5 dark:bg-gray-800 text-center">
+      <div className="flex flex-wrap justify-center items-center mx-auto max-w-screen-xl">
+        <a href="https://nepalingo.com" className="flex items-center">
+          <img src={logo} alt="Nepalingo Logo" className="h-24" />
+        </a>
+        {/* TODO: Sign out logic */}
```
Comment (christikaes) [inline nepalingo-web/src/components/Header.tsx:11]: why is this comment needed?

**Review by christikaes — APPROVED** [review]
Thanks for putting this together! I left some comments but they can be fixed in a followup PR

File: nepalingo-web/src/components/Header.tsx (hunk 2 above)
Comment (binamkayastha) [reply nepalingo-web/src/components/Header.tsx]: probably not needed, removing

**Pull request merged**